File: C/terreno.c
#include <stdio.h>
#include <locale.h>

double largura, comprimento, valor;
double area = 0;
double preco = 0;

int main() {
    setlocale(LC_ALL, "Portuguese");

    printf("Digite a largura do terreno: ");
    scanf("%lf", &largura);
    printf("Digite o comprimento do terrero: ");
    scanf("%lf", &comprimento);
    printf("Digite o valor do metro quadrado: ");
    scanf("%lf", &valor);

    area = largura * comprimento;
    preco = area * valor;

    printf("\n");
    printf("Área do terreno = %.2lf \n", area);
    printf("Preço do terreno = %.2lf \n", preco);
}

File: Python/terreno.py
largura = float(input("Digite a largura do terreno: "))
comprimento = float(input("Digite o comprimento do terreno: "))
preco = float(input("Digite o valor do metro quadrado: "))

area = largura * comprimento
valor = area * preco

print(f"Área do terreno: {area:.2f}"), print(f"Preço do terreno: {valor:.2f}")